catalog/imaging/imaging-transformer-nitf/src/test/java/org/codice/alliance/transformer/nitf/image/PreStoragePluginMultiTest.java

```diff
@@ -159,7 +161,12 @@ private void validateNitf(ContentItem contentItem, Metacard metacard) {
     private void makeNitf(ContentItem contentItem, Metacard metacard) throws IOException {
         when(contentItem.getMetacard()).thenReturn(metacard);
         when(contentItem.getId()).thenReturn("101ABC");
-        when(contentItem.getInputStream()).thenReturn(getInputStream(GEO_NITF));
+        when(contentItem.getInputStream()).thenAnswer(new Answer<InputStream>() {
```

Comment:
I think you can do `when(contentItem.getInputStream()).thenAnswer(invocationOnMock -> getInputStream(GEO_NITF));`